packages/forma-36-website/package.json

```diff
@@ -6,18 +6,30 @@
   "dependencies": {
     "@contentful/forma-36-fcss": "^0.0.13",
     "@contentful/forma-36-react-components": "^2.9.5",
+    "@contentful/forma-36-tokens": "^0.2.0",
+    "@mdx-js/mdx": "^0.17.0",
+    "@mdx-js/tag": "^0.17.0",
+    "emotion": "9.2.12",
```

Comment:
instead of using emotion directly you can use this [plugin](https://www.gatsbyjs.org/packages/gatsby-plugin-emotion/) it will help with the server rendering stuff and remove the jumpy effect when the page is loading.

Instead use this 

```
npm install --save gatsby-plugin-emotion @emotion/core @emotion/styled
```

and add the plugin to your gatsby-config

```
{
      resolve: `gatsby-plugin-emotion`,
      options: {
        // Accepts all options defined by `babel-plugin-emotion` plugin.
      },
    },
```